package/src/components/Chip/Chip.js

```diff
@@ -3,6 +3,13 @@ import PropTypes from "prop-types";
 import { Chip as MuiChip, makeStyles } from "@material-ui/core";
 
 const useStyles = makeStyles((theme) => ({
+  // Our chips are a little smaller, overriding medium and small sizes
+  root: {
+    height: "30px"
+  },
+  sizeSmall: {
+    height: "28px"
```

Comment:
Small style thing, you should use numbers rather than strings when it's only going to be 1 value like for width/height/top/bottom/etc...

Reply:
fixed.